regression-tests.auth-py/test_LuaRecords.py

```diff
@@ -55,6 +55,7 @@ class TestLuaRecords(AuthTest):
 all.noneup.ifportup          3600 IN LUA  A     "ifportup(8080, {{'192.168.42.21', '192.168.21.42'}}, {{ backupSelector='all' }})"
 
 whashed.example.org.         3600 IN LUA  A     "pickwhashed({{ {{15, '1.2.3.4'}}, {{42, '4.3.2.1'}} }})"
+whashedzero.example.org.     3600 IN LUA  A     "pickwhashed({{ {{15, '1.2.3.4'}}, {{0, '4.3.2.1'}} }})"
```

Comment:
You'll need to add `whashedzero` to `expect.txt` (alternatively you could exclude `^regression-tests\.auth-py/test_LuaRecords\.py$` to `exclude.txt`) 
https://github.com/jsoref/pdns/commit/f08ce8ad14a3070ddf5893ae267289fae0912779#commitcomment-39229497